lib/storage.hh

```diff
@@ -393,7 +403,7 @@ public:
 
     Byte ** get_raw_tables()
     {
-        return _counts;
+        return (Byte **)_counts;
```

Comment:
Not sure what to do about this. What is the use case for `get_raw_tables()` beyond nosy devs wanting to peak inside?